.github/workflows/vectorlint.yml

```diff
@@ -0,0 +1,27 @@
+name: Vector Lint
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '15 */12 * * *'
```

Comment:
Is there anything non-hermetic that requires the scheduled runs? I find GHA often fail for no good reason, and cron runs generate false fails.

Reply:
Removed.